packages/web-app-files/src/helpers/resource/conflictHandling/conflictDialog.ts

```diff
@@ -9,6 +9,7 @@ interface FileConflict {
 }
 
 export class ConflictDialog {
+  /* eslint-disable */
```

Comment:
please also add the `eslint-enable` statement after the constructor. You can disable specific rules (please do), disabling every eslint check is not recommended.